README.md

```diff
@@ -2,4 +2,10 @@
 
 These are software testing projects done during the course of my Software Testing class at SKCET. The projects are done using Selenium.
 
+### Prerequirements
```

Comment:
Add an emoji to it

Reply:
Trying using which is related to pre-requirments

Reply:
okay I will be working on it